### DISCUSSION TITLE
V23 软件源缺少open-vm-tool 包

### DISCUSSION THREAD

**Discussion by uestc-alayi** [linuxdeepin/developer-center#4359] [Q&A | 问答板块]
V23 软件源缺少open-vm-tool 包

**Comment by BLumia** [comment]
https://github.com/linuxdeepin/developer-center/issues/4020 应该有了才对。如果是 alpha2 而没在内测的话，需要进内测或者等 beta 发布。

cc @UTsweetyfish
